src/components/CartItem/CartItem.tsx

```diff
@@ -0,0 +1,13 @@
+interface CartItemProps {
+  image: string;
+  title: string;
+  price: number;
+  quantity: number;
+  stock: number;
```

Comment:
this is not correct. 
Firstly, CartItem should be an separate interface.
```
interface CartItem {
  productId: string;
  quantity: number;
}
```
For cart item props, it will include product data as well
interface CartItemProps {
  cartItem: CartItem;
  product: Product;
}
```